packages/xo-server-transport-nagios/src/index.js

```diff
@@ -2,11 +2,23 @@ import crc32 from 'buffer-crc32'
 import net from 'net'
 import { Buffer } from 'buffer'
 import { createLogger } from '@xen-orchestra/log'
+// eslint-disable-next-line n/no-extraneous-import
```

Comment:
Why?

Don't disable an error unless you really understand what it means and why it's not relevant.

In this case, I believe it really reveals a problem in this PR: the dependency is missing from `package.json`.

Reply:
You're right. Adding the dependency has removed the error.